internal/usergroup/parser_test.go

```diff
@@ -0,0 +1,62 @@
+package usergroup
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/docker/docker/pkg/idtools"
+	"github.com/moby/sys/user"
+	"gotest.tools/v3/assert"
+)
+
+func TestCreateIDMapOrder(t *testing.T) {
+	subidRanges := []user.SubID{
+		{Name: "", SubID: 100000, Count: 1000},
+		{Name: "", SubID: 1000, Count: 1},
+	}
+
+	idMap := createIDMap(subidRanges)
+	assert.DeepEqual(t, idMap, []idtools.IDMap{
+		{
+			ContainerID: 0,
+			HostID:      100000,
+			Size:        1000,
+		},
+		{
+			ContainerID: 1000,
+			HostID:      1000,
+			Size:        1,
+		},
+	})
+}
```

Comment:
Looks like this is no longer testing anything after the rewrite to moby/sys/user